packages/builder/test/internal/core/config/fixtures-account.ts

```diff
@@ -0,0 +1,6 @@
+import { Account, mnemonicToSecretKey } from "algosdk";
```

Comment:
We have our test projects in a separate directory. I think it's a good idea to separate mock objects too.

Reply:
OK, I will create a `mock` subdirectory.